libs/application/templates/healthcare-work-permit/src/forms/HealthcareWorkPermitForm/personalSection.ts

```diff
@@ -0,0 +1,88 @@
+import {
+  buildDescriptionField,
+  buildMultiField,
+  buildPhoneField,
+  buildSection,
+  buildTextField,
+} from '@island.is/application/core'
+import { personal } from '../../lib/messages'
+import { Application } from '@island.is/api/schema'
+import { formatDate } from '../../utils'
+
+export const PersonalSection = buildSection({
```

Comment:
Use the shared applicantInformationMultiField ?

Reply:
Wasn't aware of this, but it seems to not have the same fields we are using for the personalSection